src/cascadia/TerminalSettingsModel/ActionArgs.idl

```diff
@@ -234,16 +234,13 @@ namespace Microsoft.Terminal.Settings.Model
         NewTerminalArgs TerminalArgs { get; };
     };
 
-    unsealed runtimeclass SwitchToAdjacentTabArgs : IActionArgs
+    [default_interface] runtimeclass PrevTabArgs : IActionArgs

```

Comment:
I just remembered... if I recall correctly (@zadjii-msft can keep me honest!) there's a way to have one argument type apply to two different actions -- part of why we made the action interface take arguments separately. Since these are just the same action, we might want to keep the root node (SwitchToAdjacentTabArgs) and delete the _leaves_. If we can!

Reply:
Oh, but they're separate for the purposes of generating the _command name_. Aaargh

Reply:
I may file followup work to make ActionAndArgs::GenerateName more flexible.

```cpp
// If the args class supports appending to an existing command name, do that instead
if (auto appendName{ _Args.try_as<ICanAppendCommandName>() }) {
    auto otherName = GeneratedActionNames.find(_Action);
    return appendName.AppendName(otherName);
}
```

one of these days

Reply:
I can open a follow up for this :blush: 
Any additional blockers here?

Reply:
No blockers! It would be very cool if we consolidated all the action types that only differ based on name :smile: in the followup of course